modules/ecs-cluster/README.md

```diff
@@ -9,13 +9,13 @@ Define cluster:
 ```tf
 module "ecs_cluster" {
   source  = "Selleo/backend/aws//modules/ecs-cluster"
-  version = "0.3.0"
+  version = "0.3.1"
```

Comment:
Maybe we should consider moving it to `0.4.0`. I think change to cloud init is a big enough one to warrant this, wdyt?

Reply:
yes, let's do 0.4